libraries/botbuilder/tests/streaming/botFrameworkAdapterStreaming.test.js

```diff
@@ -124,7 +124,7 @@ describe('BotFrameworkAdapter Streaming tests', () => {
                 });
             } catch (err) {
                 expect(err.statusCode).to.equal(StatusCodes.UNAUTHORIZED);
-                expect(err.message).to.equal('Unauthorized. No valid identity.');
+                expect(err.message).to.equal('Unauthorized. Is not authenticated');
```

Comment:
Prior to this change, this test passes when test are run from `root` to `botbuilder\tests`. If they were run from `botbuilder\tests\streaming`, this test would fail because, when run solo, it lacked the environment variable for the ChannelService. When the tests above ran, they set the ChannelService to `'undefined'` which, because it is a string, is not falsy.

Therefore, I **believe** this change is appropriate, but am willing to concede if somebody has a strong argument, otherwise.

Reply:
I agree with Michael. His change should be the correct error message, because the the test uses `'donttrustme'`, which eventually becomes `identity` once in the `*ChannelValidation` layer of authentication.

```ts

        if (!identity) {
            // No valid identity. Not Authorized.
            throw new AuthenticationError('Unauthorized. No valid identity.', StatusCodes.UNAUTHORIZED);
        }

        if (!identity.isAuthenticated) {
            // The token is in some way invalid. Not Authorized.
            throw new AuthenticationError('Unauthorized. Is not authenticated', StatusCodes.UNAUTHORIZED);
        }
```

`identity` *is* provided, it's just not validated to be an authenticated token, therefore should throw `Unauthorized. Is not authenticated` as the err message